src/schema/objects/suffixes.yaml

```diff
@@ -286,16 +292,23 @@ RB1map:
   unit: arbitrary
 S0map:
   value: S0map
-  display_name: Observed signal amplitude (S0) image
+  display_name: Projected baseline signal amplitude (S0) image
```

Comment:
Projected -> Predicted? Estimated?

Reply:
Don't feel particularly strongly for any choice.

Reply:
Agreed. I'm fine if others feel like "projected" is the right.